okrs/2019-q3.md

```diff
@@ -0,0 +1,19 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+
+| Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | -------- | ----------- | --------------- | ----------- |
+| IPLD Schema for UnixFSv2 | @mikeal | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Spec for additional attributes in UnixFSv1 | @mikeal | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Complete selector work necessary for GraphSync | @hannahhoward | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| Schema based codegen works in go-ipld-prime | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
```

Comment:
I'm tempted to heap a bunch more conditionals and break this down in the smaller bits, because I doubt this will be 100% in a quarter -- but aiming for MVP (with less than 100% of all minor features filled in)?  Yes.  And this is kind of the hub and drive of all the individual efforts, so.  :+1:

Reply:
I tweaked the language in the KR to apply to all of this work through the end of the year and noted after the KR definitions what we’d expect to see this quarter. Let me know if this sounds doable.